<launch>
<!-- 
        ros2 bag play ros2 bag play FIU_BBC_Pier_Rat/FIU_BBC_Pier_Rat_0.db3 
        -\-rate 1.0   
        -\-clock   
        -\-start-paused 
        -\-topics /surveyor/camera/image/compressed /surveyor/odom /surveyor/gps_fix -->
    <node
        pkg="ratslam"
        exec="ratslam_lv"
        name="ratslam_view_template"
        output="screen"
        args="--ros-args --remap _image_transport:=compressed">
        <param name="use_sim_time" value="true" />
        <param name="topic_root" value="surveyor" />
        <param name="media_path" value="$(find-pkg-share ratslam)/media" />
        <param name="image_file" value="irat_sm.tga" />
        <param from="$(find-pkg-share ratslam)/config/config_surveyor_BBC_Pier.yaml" />
    </node>

    <node
        pkg="ratslam"
        exec="ratslam_pc"
        name="ratslam_pose_cells"
        output="screen"
        args="--ros-args --remap _image_transport:=compressed">
        <param name="use_sim_time" value="true" />
        <param name="topic_root" value="surveyor" />
        <param name="media_path" value="$(find-pkg-share ratslam)/media" />
        <param name="image_file" value="irat_sm.tga" />
        <param from="$(find-pkg-share ratslam)/config/config_surveyor_BBC_Pier.yaml" />
    </node>

    <node
        pkg="ratslam"
        exec="ratslam_em"
        name="ratslam_experience_map"
        output="screen"
        args="--ros-args --remap _image_transport:=compressed">
        <param name="use_sim_time" value="true" />
        <param name="topic_root" value="surveyor" />
        <param name="media_path" value="$(find-pkg-share ratslam)/media" />
        <param name="image_file" value="irat_sm.tga" />
        <param from="$(find-pkg-share ratslam)/config/config_surveyor_BBC_Pier.yaml" />
    </node>      
            
<!--  
    <arg name="config_file" default="$(find-pkg-share ratslam)/config/config_irataus.yaml"/>
    <node
        pkg="ratslam"
        exec="ratslam_lv"
        name="RatSLAMLocalViewCells"
		namespace="irat_red"
        output="screen">
    </node> -->
</launch>

